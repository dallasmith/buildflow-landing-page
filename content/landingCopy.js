export const landingCopy = {
  meta: {
    title: "BuildFlow - Stop Chasing Subs. Start Building.",
    lang: "en"
  },
  header: {
    logo: "BuildFlowâ„¢",
    nav: {
      howItWorks: "How it Works",
      features: "Features",
      faq: "FAQ"
    },
    cta: "ðŸ”’ Reserve Your Spot â€” $10"
  },
  hero: {
    headline: "Stop Chasing Subs. Start Building.",
    subtitle: "BuildFlow automates communication with your subcontractors so you can focus on what matters - getting the job done. No more 6 AM texts or missed deadlines.",
    cta: {
      primary: "ðŸ”’ Reserve Your Spot â€” $10",
      trustText: "Fully refundable. No strings attached."
    }
  },
  problem: {
    headline: "Doing More Babysitting Than Building?",
    items: [
      {
        title: "Your Phone is the Job",
        description: "15+ calls a day just to check if subs are still coming. \"You still good for Tuesday?\" becomes your full-time job."
      },
      {
        title: "Missed Deadlines",
        description: "Subs ghost you, delays cascade, and you're stuck explaining to clients why their project is behind schedule."
      },
      {
        title: "Lost Revenue",
        description: "Every hour spent texting subs is an hour lost on site or finding new projects."
      }
    ]
  },
  appScreenshot: {
    headline: "Know What's Locked In â€” At a Glance",
    subtitle: "No more \"Did he confirm?\" texts. See exactly who's coming, who's pending, and who needs backup."
  },
  noteFromBuilder: {
    headline: "Built on the Job Site - By Someone Still in It",
    content: [
      "I came from the job site. I was the one loading materials, sweeping floors, and doing whatever needed to get done. 6 months later, I was running multiple projects, trying to keep subs lined up and jobs moving. Between tracking who confirmed, following up on no-shows, and explaining delays to clients, my whole day was spent managing chaos.",
      "That's when I started thinking: There's got to be a better way.",
      "That's why I built BuildFlow. It's not some bloated platform with a thousand tabs â€” just a simple tool for people like you and me: managing multiple jobs, juggling trades, and trying to keep the day on track. BuildFlow helps you stay in control, keep your subs aligned, and get through the chaos with fewer headaches."
    ],
    signature: {
      name: "â€” Dallas",
      title: "Superintendent & Creator of BuildFlow"
    }
  },
  howItWorks: {
    headline: "Less Phone Time. More Build Time.",
    steps: [
      {
        number: "1",
        title: "Create Your Project",
        description: "Set up your project details and assign tasks to your subs (primary + backup)."
      },
      {
        number: "2",
        title: "BuildFlow Texts Subs",
        description: "Automatic SMS with job details. Subs just reply \"Y\" or \"N\" - no apps needed."
      },
      {
        number: "3",
        title: "Smart Reminders & Backup",
        description: "48h and 24h reminders. Switch to a backup sub with one tap if needed."
      }
    ],
    cta: {
      primary: "ðŸ”’ Reserve Your Spot â€” $10",
      trustText: "Fully refundable. No strings attached."
    }
  },
  projectsScreenshot: {
    headline: "Multiple Jobs, Zero Chaos",
    subtitle: "Whether you've got two projects or ten, BuildFlow helps you track them all â€” clearly and simply."
  },
  textConversations: {
    headline: "Real Conversations, Real Simple",
    subtitle: "Your subs get texts like always â€” but now they're smart and automated."
  },
  features: {
    headline: "Built for Builders Who Actually Build",
    items: [
      {
        title: "Text-Based Scheduling",
        description: "No sub apps or training. They get your messages like always - but now it's automated."
      },
      {
        title: "Smart Reminders",
        description: "48h and 24h reminders sent automatically."
      },
      {
        title: "Backup Subs",
        description: "Assign fallbacks. Switch to a backup sub with one tap when needed."
      },
      {
        title: "Targeted Notifications",
        description: "Only affected trades get delay notifications - not everyone on the project."
      },
      {
        title: "Real-Time Updates",
        description: "See who's confirmed, who's pending, and who's declined at a glance."
      },
      {
        title: "Zero Training",
        description: "Your subs already know how to text. No new apps, no learning curve."
      }
    ]
  },
  beforeAfter: {
    headline: "Before vs. After BuildFlow",
    before: {
      title: "ðŸ”´ The Old Way",
      items: [
        "\"You still good for Thursday?\"",
        "\"Did anyone remind the framer?\"",
        "\"He ghosted. Now what?\"",
        "15+ calls a day",
        "Missed deadlines",
        "Stressed clients"
      ]
    },
    after: {
      title: "âœ… With BuildFlow",
      items: [
        "Subs already confirmed days ago",
        "Smart reminders sent automatically",
        "Backup sub already invited",
        "Maybe 2 calls a day",
        "Projects stay on schedule",
        "Happy clients"
      ]
    }
  },
  ctaSection: {
    headline: "Get in Early. Stay Ahead.",
    description: "Lock in your spot for just $10. You'll get:",
    benefits: [
      "âœ… First access to the beta",
      "âœ… Lifetime discount",
      "âœ… Priority support"
    ],
    cta: "ðŸ”’ Reserve Your Spot â€” $10",
    trustText: "Refund anytime before launch. No risk."
  },
  faq: {
    headline: "Frequently Asked Questions",
    items: [
      {
        question: "Do I need to install anything?",
        answer: "Yes â€” you'll install the BuildFlow app. But your subs won't. They just get texts from you like they already do. It's seamless for them."
      },
      {
        question: "What happens after I pay?",
        answer: "You're added to the early access crew. When beta opens, you'll be first in line â€” and locked in at a discount."
      },
      {
        question: "Why charge $10?",
        answer: "It filters out tire-kickers and helps us prioritize builders who are serious. If it's not a fit, you'll get your $10 back."
      },
      {
        question: "Will this bother my subs?",
        answer: "Nope. They're already getting texts from you. This just makes them smarter, more consistent, and less likely to be forgotten."
      }
    ]
  },
  finalCta: {
    headline: "Ready to Stop Chasing Subs?",
    description: "Join the builders who are already ahead of the game.",
    cta: "ðŸ”’ Reserve Your Spot â€” $10"
  },
  footer: {
    buildflow: {
      title: "BuildFlow",
      description: "Built by builders, for builders. Stop chasing subs. Start building.",
      socialLinks: [
        { label: "Twitter", href: "#", ariaLabel: "Twitter" },
        { label: "LinkedIn", href: "#", ariaLabel: "LinkedIn" },
        { label: "Instagram", href: "#", ariaLabel: "Instagram" }
      ]
    },
    product: {
      title: "Product",
      links: [
        { label: "How it Works", href: "#how-it-works" },
        { label: "Features", href: "#features" },
        { label: "FAQ", href: "#faq" }
      ]
    },
    support: {
      title: "Support",
      links: [
        { label: "Contact Us", href: "mailto:hello@buildflow.com" },
        { label: "Help Center", href: "#faq" },
        { label: "Status", href: "#" }
      ]
    },
    legal: {
      title: "Legal",
      links: [
        { label: "Privacy Policy", href: "privacy.html" },
        { label: "Terms of Service", href: "terms.html" },
        { label: "Cookie Policy", href: "cookies.html" }
      ]
    },
    bottom: "Â© 2024 BuildFlow. All rights reserved."
  }
} as const;
