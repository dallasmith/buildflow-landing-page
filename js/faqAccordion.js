/**
 * FAQ Accordion Module - Accessible, keyboard-friendly accordion
 * Supports Enter/Space to toggle, Arrow keys for navigation
 */

export function initFaqAccordion(containerSelector = '#faq') {
  const container = document.querySelector(containerSelector);
  if (!container) {
    console.warn(`FAQAccordion: Container ${containerSelector} not found`);
    return;
  }

  // Find all FAQ items
  const faqItems = container.querySelectorAll('.faq-item');
  if (faqItems.length === 0) {
    console.warn('FAQAccordion: No FAQ items found');
    return;
  }

  console.log(`FAQAccordion: Found ${faqItems.length} items, initializing...`);

  // Transform each FAQ item to accordion structure
  faqItems.forEach((item, index) => {
    const question = item.querySelector('h3');
    const answer = item.querySelector('p');
    
    if (!question || !answer) {
      console.warn(`FAQAccordion: Invalid structure for item ${index}`);
      return;
    }

    // Get the text content
    const questionText = question.textContent;
    const answerText = answer.textContent;

    // Create new accordion structure
    const toggleButton = document.createElement('button');
    toggleButton.className = 'faq-toggle';
    toggleButton.setAttribute('aria-expanded', 'false');
    toggleButton.setAttribute('aria-controls', `faq-a-${index}`);
    toggleButton.id = `faq-q-${index}`;
    toggleButton.textContent = questionText;

    const panel = document.createElement('div');
    panel.className = 'faq-panel';
    panel.id = `faq-a-${index}`;
    panel.setAttribute('role', 'region');
    panel.setAttribute('aria-labelledby', `faq-q-${index}`);
    panel.setAttribute('hidden', '');
    panel.textContent = answerText;

    // Clear the original content and add new structure
    item.innerHTML = '';
    item.appendChild(toggleButton);
    item.appendChild(panel);

    // Add click handler
    toggleButton.addEventListener('click', () => togglePanel(toggleButton, panel));

    // Add keyboard handlers
    toggleButton.addEventListener('keydown', (e) => handleKeydown(e, index, faqItems.length));
  });

  // Function to toggle panel
  function togglePanel(button, panel) {
    const isExpanded = button.getAttribute('aria-expanded') === 'true';
    const prefersReducedMotion = window.matchMedia('(prefers-reduced-motion: reduce)').matches;

    if (isExpanded) {
      button.setAttribute('aria-expanded', 'false');
      panel.setAttribute('hidden', '');
    } else {
      button.setAttribute('aria-expanded', 'true');
      panel.removeAttribute('hidden');
    }

    // Add visual feedback
    if (!prefersReducedMotion) {
      panel.style.transition = 'all 0.3s ease-in-out';
    }
  }

  // Function to handle keyboard navigation
  function handleKeydown(event, currentIndex, totalItems) {
    switch (event.key) {
      case 'Enter':
      case ' ':
        event.preventDefault();
        const currentButton = event.target;
        const currentPanel = document.getElementById(currentButton.getAttribute('aria-controls'));
        togglePanel(currentButton, currentPanel);
        break;

      case 'ArrowDown':
        event.preventDefault();
        const nextIndex = (currentIndex + 1) % totalItems;
        const nextButton = document.getElementById(`faq-q-${nextIndex}`);
        if (nextButton) {
          nextButton.focus();
        }
        break;

      case 'ArrowUp':
        event.preventDefault();
        const prevIndex = currentIndex === 0 ? totalItems - 1 : currentIndex - 1;
        const prevButton = document.getElementById(`faq-q-${prevIndex}`);
        if (prevButton) {
          prevButton.focus();
        }
        break;
    }
  }

  // Add CSS for focus-visible (if not already present)
  if (!document.querySelector('#faq-accordion-styles')) {
    const style = document.createElement('style');
    style.id = 'faq-accordion-styles';
    style.textContent = `
      .faq-toggle:focus-visible {
        outline: 2px solid #015ef4;
        outline-offset: 2px;
      }
    `;
    document.head.appendChild(style);
  }

  console.log(`FAQAccordion: Initialized with ${faqItems.length} items`);
}


