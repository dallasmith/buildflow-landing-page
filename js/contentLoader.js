// Content loader for BuildFlow landing page
// This script loads content from landingCopy.js and applies it to the HTML

import { landingCopy } from '../content/landingCopy.js';

// Function to safely set text content
function setTextContent(selector, text) {
  const element = document.querySelector(selector);
  if (element && text !== undefined) {
    element.textContent = text;
  }
}

// Function to safely set HTML content
function setHTMLContent(selector, html) {
  const element = document.querySelector(selector);
  if (element && html !== undefined) {
    element.innerHTML = html;
  }
}

// Function to populate header content
function populateHeader() {
  setTextContent('.logo', landingCopy.header.logo);
  setTextContent('nav a[href="#how-it-works"]', landingCopy.header.nav.howItWorks);
  setTextContent('nav a[href="#features"]', landingCopy.header.nav.features);
  setTextContent('nav a[href="#faq"]', landingCopy.header.nav.faq);
  setTextContent('.header-cta', landingCopy.header.cta);
}

// Function to populate hero section
function populateHero() {
  setTextContent('#hero h1', landingCopy.hero.headline);
  setHTMLContent('#hero .hero-subtitle', landingCopy.hero.subtitleHTML);
  setTextContent('#hero .primary-cta', landingCopy.hero.cta.primary);
  setHTMLContent('#hero .trust-text', landingCopy.hero.trustHTML);
}

// Function to populate problem section
function populateProblem() {
  setTextContent('#problem h2', landingCopy.problem.headline);
  
  const problemItems = document.querySelectorAll('#problem .problem-item');
  landingCopy.problem.items.forEach((item, index) => {
    if (problemItems[index]) {
      setTextContent(`#problem .problem-item:nth-child(${index + 1}) h3`, item.title);
      setTextContent(`#problem .problem-item:nth-child(${index + 1}) p`, item.description);
    }
  });
}

// Function to populate app screenshot section
function populateAppScreenshot() {
  setTextContent('#app-screenshot h2', landingCopy.appScreenshot.headline);
  setTextContent('#app-screenshot .screenshot-subtitle', landingCopy.appScreenshot.subtitle);
}

// Function to populate note from builder section
function populateNoteFromBuilder() {
  setTextContent('#note-from-builder h2', landingCopy.noteFromBuilder.headline);
  
  const contentParagraphs = document.querySelectorAll('#note-from-builder .builder-content p');
  landingCopy.noteFromBuilder.content.forEach((paragraph, index) => {
    if (contentParagraphs[index]) {
      setTextContent(`#note-from-builder .builder-content p:nth-child(${index + 1})`, paragraph);
    }
  });
  
  setTextContent('#note-from-builder .builder-signature strong', landingCopy.noteFromBuilder.signature.name);
  setTextContent('#note-from-builder .builder-signature span', landingCopy.noteFromBuilder.signature.title);
}

// Function to populate how it works section
function populateHowItWorks() {
  setTextContent('#how-it-works h2', landingCopy.howItWorks.headline);
  
  const steps = document.querySelectorAll('#how-it-works .step');
  landingCopy.howItWorks.steps.forEach((step, index) => {
    if (steps[index]) {
      setTextContent(`#how-it-works .step:nth-child(${index + 1}) .step-number`, step.number);
      setTextContent(`#how-it-works .step:nth-child(${index + 1}) h3`, step.title);
      setTextContent(`#how-it-works .step:nth-child(${index + 1}) p`, step.description);
    }
  });
  
  setTextContent('#how-it-works .primary-cta', landingCopy.howItWorks.cta.primary);
  setTextContent('#how-it-works .trust-text', landingCopy.howItWorks.cta.trustText);
}

// Function to populate projects screenshot section
function populateProjectsScreenshot() {
  setTextContent('#projects-screenshot h2', landingCopy.projectsScreenshot.headline);
  setTextContent('#projects-screenshot .screenshot-subtitle', landingCopy.projectsScreenshot.subtitle);
}

// Function to populate text conversations section
function populateTextConversations() {
  setTextContent('#text-conversations h2', landingCopy.textConversations.headline);
  setTextContent('#text-conversations .screenshot-subtitle', landingCopy.textConversations.subtitle);
}

// Function to populate features section
function populateFeatures() {
  setTextContent('#features h2', landingCopy.features.headline);
  
  const featureItems = document.querySelectorAll('#features .feature');
  landingCopy.features.items.forEach((feature, index) => {
    if (featureItems[index]) {
      setTextContent(`#features .feature:nth-child(${index + 1}) h3`, feature.title);
      setTextContent(`#features .feature:nth-child(${index + 1}) p`, feature.description);
    }
  });
}

// Function to populate before/after section
function populateBeforeAfter() {
  setTextContent('#before-after h2', landingCopy.beforeAfter.headline);
  
  setTextContent('#before-after .before h3', landingCopy.beforeAfter.before.title);
  setTextContent('#before-after .after h3', landingCopy.beforeAfter.after.title);
  
  const beforeItems = document.querySelectorAll('#before-after .before ul li');
  landingCopy.beforeAfter.before.items.forEach((item, index) => {
    if (beforeItems[index]) {
      setTextContent(`#before-after .before ul li:nth-child(${index + 1})`, item);
    }
  });
  
  const afterItems = document.querySelectorAll('#before-after .after ul li');
  landingCopy.beforeAfter.after.items.forEach((item, index) => {
    if (afterItems[index]) {
      setTextContent(`#before-after .after ul li:nth-child(${index + 1})`, item);
    }
  });
}

// Function to populate CTA section
function populateCtaSection() {
  setTextContent('#cta-section h2', landingCopy.ctaSection.headline);
  setTextContent('#cta-section p', landingCopy.ctaSection.description);
  
  const benefits = document.querySelectorAll('#cta-section .benefit');
  landingCopy.ctaSection.benefits.forEach((benefit, index) => {
    if (benefits[index]) {
      setTextContent(`#cta-section .benefit:nth-child(${index + 1})`, benefit);
    }
  });
  
  setTextContent('#cta-section .primary-cta', landingCopy.ctaSection.cta);
  setTextContent('#cta-section .trust-text', landingCopy.ctaSection.trustText);
}

// Function to populate FAQ section
function populateFaq() {
  setTextContent('#faq h2', landingCopy.faq.headline);
  
  const faqItems = document.querySelectorAll('#faq .faq-item');
  landingCopy.faq.items.forEach((faq, index) => {
    if (faqItems[index]) {
      setTextContent(`#faq .faq-item:nth-child(${index + 1}) h3`, faq.question);
      setTextContent(`#faq .faq-item:nth-child(${index + 1}) p`, faq.answer);
    }
  });
}

// Function to populate final CTA section
function populateFinalCta() {
  setTextContent('#final-cta h2', landingCopy.finalCta.headline);
  setTextContent('#final-cta p', landingCopy.finalCta.description);
  setTextContent('#final-cta .primary-cta', landingCopy.finalCta.cta);
}

// Function to populate footer
function populateFooter() {
  setTextContent('footer .footer-section:nth-child(1) h4', landingCopy.footer.buildflow.title);
  setTextContent('footer .footer-section:nth-child(1) p', landingCopy.footer.buildflow.description);
  
  setTextContent('footer .footer-section:nth-child(2) h4', landingCopy.footer.product.title);
  setTextContent('footer .footer-section:nth-child(3) h4', landingCopy.footer.support.title);
  setTextContent('footer .footer-section:nth-child(4) h4', landingCopy.footer.legal.title);
  
  setTextContent('footer .footer-bottom p', landingCopy.footer.bottom);
}

// Main function to populate all content
function populateAllContent() {
  try {
    populateHeader();
    populateHero();
    populateProblem();
    populateAppScreenshot();
    populateNoteFromBuilder();
    populateHowItWorks();
    populateProjectsScreenshot();
    populateTextConversations();
    populateFeatures();
    populateBeforeAfter();
    populateCtaSection();
    populateFaq();
    populateFinalCta();
    populateFooter();
    
    console.log('BuildFlow landing page content loaded successfully');
  } catch (error) {
    console.error('Error loading BuildFlow content:', error);
  }
}

// Export the main function
export { populateAllContent };

// Auto-populate when DOM is ready
if (document.readyState === 'loading') {
  document.addEventListener('DOMContentLoaded', populateAllContent);
} else {
  populateAllContent();
}
