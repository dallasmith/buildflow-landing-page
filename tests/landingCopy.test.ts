import { describe, it, expect } from 'vitest';
import { landingCopy } from '../content/landingCopy.js';

/**
 * SNAPSHOT TEST FOR LANDING PAGE COPY
 * 
 * ⚠️  IMPORTANT: This test prevents accidental copy changes
 * 
 * If you need to update copy:
 * 1. Get explicit approval from business owner
 * 2. Update the content in content/landingCopy.ts
 * 3. Run: npm run test -- --update-snapshots
 * 4. Verify the changes are correct
 * 5. Commit both content changes AND updated snapshot
 * 
 * NEVER update snapshots without business approval!
 */

describe('landingCopy', () => {
  it('should maintain exact copy consistency', () => {
    // This snapshot captures the EXACT current copy
    // Any text changes will cause this test to fail
    expect(landingCopy).toMatchSnapshot();
  });

  it('should have all required sections', () => {
    // Verify structure integrity
    expect(landingCopy).toHaveProperty('hero');
    expect(landingCopy).toHaveProperty('problem');
    expect(landingCopy).toHaveProperty('appScreenshot');
    expect(landingCopy).toHaveProperty('howItWorks');
    expect(landingCopy).toHaveProperty('projectsScreenshot');
    expect(landingCopy).toHaveProperty('textConversations');
    expect(landingCopy).toHaveProperty('features');
    expect(landingCopy).toHaveProperty('beforeAfter');
    expect(landingCopy).toHaveProperty('ctaSection');
    expect(landingCopy).toHaveProperty('faq');
    expect(landingCopy).toHaveProperty('finalCta');
  });

  it('should have non-empty content in all sections', () => {
    // Ensure no empty strings or missing content
    const checkContent = (obj: Record<string, unknown>, path: string = '') => {
      for (const [key, value] of Object.entries(obj)) {
        const currentPath = path ? `${path}.${key}` : key;
        
        if (typeof value === 'string') {
          expect(value.trim()).not.toBe('', `${currentPath} should not be empty`);
          expect(value.length).toBeGreaterThan(0, `${currentPath} should have content`);
        } else if (typeof value === 'object' && value !== null && !Array.isArray(value)) {
          checkContent(value, currentPath);
        } else if (Array.isArray(value)) {
          expect(value.length).toBeGreaterThan(0, `${currentPath} should not be empty array`);
          value.forEach((item, index) => {
            if (typeof item === 'object' && item !== null) {
              checkContent(item, `${currentPath}[${index}]`);
            }
          });
        }
      }
    };

    checkContent(landingCopy);
  });

  it('should maintain consistent BuildFlow branding', () => {
    // Ensure BuildFlow is consistently referenced
    const buildFlowReferences = JSON.stringify(landingCopy).match(/BuildFlow/g);
    expect(buildFlowReferences).toBeTruthy();
    expect(buildFlowReferences!.length).toBeGreaterThan(0);
  });

  it('should have FAQ items with both questions and answers', () => {
    // Verify FAQ structure
    expect(landingCopy.faq).toHaveProperty('items');
    expect(landingCopy.faq.items).toBeInstanceOf(Array);
    expect(landingCopy.faq.items.length).toBeGreaterThan(0);
    
    landingCopy.faq.items.forEach((item, index) => {
      expect(item).toHaveProperty('question');
      expect(item).toHaveProperty('answer');
      expect(item.question.trim()).not.toBe('', `FAQ item ${index} question is empty`);
      expect(item.answer.trim()).not.toBe('', `FAQ item ${index} answer is empty`);
    });
  });
});
