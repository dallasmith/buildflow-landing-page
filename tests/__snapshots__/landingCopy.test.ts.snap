// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`landingCopy > should maintain exact copy consistency 1`] = `
{
  "appScreenshot": {
    "headline": "Know What's Locked In â€” At a Glance",
    "subtitle": "Save hours every week coordinating your subcontractors.",
  },
  "beforeAfter": {
    "after": {
      "items": [
        "Subs already confirmed days ago",
        "Smart reminders sent automatically",
        "Backup sub already invited",
        "Far fewer calls each day",
        "Projects stay on schedule",
        "Happy clients",
      ],
      "title": "âœ… With BuildFlow",
    },
    "before": {
      "items": [
        ""You still good for Thursday?"",
        ""Did anyone remind the framer?"",
        ""He ghosted. Now what?"",
        "15+ calls a day",
        "Missed deadlines",
        "Stressed clients",
      ],
      "title": "ðŸ”´ The Old Way",
    },
    "headline": "Before vs. After BuildFlow",
  },
  "ctaSection": {
    "benefits": [
      "âœ… First access to the beta",
      "âœ… Lifetime discount",
      "âœ… Priority support",
    ],
    "cta": "ðŸ”’ Reserve Your Spot â€” $10",
    "description": "Lock in your spot for $10. You'll get:",
    "headline": "Get in Early. Stay Ahead. (Only 25 beta spots)",
    "trustText": "Refund anytime before launch. No risk.",
  },
  "faq": {
    "headline": "Frequently Asked Questions",
    "items": [
      {
        "answer": "Yesâ€”you'll install the BuildFlow app. Your subs won't. They'll get texts from you like they already do. It's seamless for them.",
        "question": "Do I need to install anything?",
      },
      {
        "answer": "You're added to the early access crew. When beta opens, you'll be first in line â€” and locked in at a discount.",
        "question": "What happens after I pay?",
      },
      {
        "answer": "It helps us prioritize builders who are serious about early access. If it's not a fit, you'll get your $10 back.",
        "question": "Why charge $10?",
      },
      {
        "answer": "No. They already get texts from youâ€”this just makes them more consistent and less likely to be forgotten.",
        "question": "Will this bother my subs?",
      },
    ],
  },
  "features": {
    "headline": "Built for Builders Who Actually Build",
    "items": [
      {
        "description": "No sub apps or training. They get your texts like alwaysâ€”now it's automated.",
        "title": "Text-Based Scheduling",
      },
      {
        "description": "48h and 24h reminders sent automatically.",
        "title": "Smart Reminders",
      },
      {
        "description": "Assign fallbacks. Switch to a backup with one tap when needed.",
        "title": "Backup Subs",
      },
      {
        "description": "Only affected trades get delay notificationsâ€”not everyone on the project.",
        "title": "Targeted Notifications",
      },
      {
        "description": "See who's confirmed, who's pending, and who's declined at a glance.",
        "title": "Real-Time Updates",
      },
      {
        "description": "Your subs already know how to text. No new apps, no learning curve.",
        "title": "Zero Training",
      },
    ],
  },
  "finalCta": {
    "cta": "ðŸ”’ Reserve Your Spot â€” $10",
    "description": "Join builders who want to get ahead.",
    "headline": "Ready to Stop Chasing Subs?",
  },
  "footer": {
    "bottom": "Â© 2024 BuildFlow. All rights reserved.",
    "buildflow": {
      "description": "Built by builders, for builders. Stop chasing subs. Start building.",
      "socialLinks": [
        {
          "ariaLabel": "Twitter",
          "href": "#",
          "label": "Twitter",
        },
        {
          "ariaLabel": "LinkedIn",
          "href": "#",
          "label": "LinkedIn",
        },
        {
          "ariaLabel": "Instagram",
          "href": "#",
          "label": "Instagram",
        },
      ],
      "title": "BuildFlow",
    },
    "legal": {
      "links": [
        {
          "href": "privacy.html",
          "label": "Privacy Policy",
        },
        {
          "href": "terms.html",
          "label": "Terms of Service",
        },
        {
          "href": "cookies.html",
          "label": "Cookie Policy",
        },
      ],
      "title": "Legal",
    },
    "product": {
      "links": [
        {
          "href": "#how-it-works",
          "label": "How it Works",
        },
        {
          "href": "#features",
          "label": "Features",
        },
        {
          "href": "#faq",
          "label": "FAQ",
        },
      ],
      "title": "Product",
    },
    "support": {
      "links": [
        {
          "href": "mailto:hello@buildflow.com",
          "label": "Contact Us",
        },
        {
          "href": "#faq",
          "label": "Help Center",
        },
        {
          "href": "#",
          "label": "Status",
        },
      ],
      "title": "Support",
    },
  },
  "header": {
    "cta": "ðŸ”’ Reserve Your Spot â€” $10",
    "logo": "BuildFlowâ„¢",
    "nav": {
      "faq": "FAQ",
      "features": "Features",
      "howItWorks": "How it Works",
    },
  },
  "hero": {
    "cta": {
      "primary": "ðŸ”’ Reserve Your Spot â€” $10",
      "trustText": "Be the first to try BuildFlow. Lock in early access for just $10 â€” fully refundable. Spots are limited â€” we're only taking 25 early users for the beta.",
    },
    "headline": "Stop Chasing Subs. Start Building.",
    "subtitle": "BuildFlow automates scheduling and follow-ups with your subcontractors so you spend less time chasing and more time building. No sub app required â€” they just get simple text reminders.",
  },
  "howItWorks": {
    "cta": {
      "primary": "ðŸ”’ Reserve Your Spot â€” $10",
      "trustText": "Fully refundable. No strings attached.",
    },
    "headline": "Less Phone Time. More Build Time.",
    "steps": [
      {
        "description": "Set your project details and assign tasks and subs (primary + backup).",
        "number": "1",
        "title": "Create Your Project",
      },
      {
        "description": "Automatic SMS with job details. Subs reply "Y" or "N"â€”no apps needed.",
        "number": "2",
        "title": "BuildFlow Texts Your Subs",
      },
      {
        "description": "48h and 24h reminders. Switch to a backup sub with one tap if needed.",
        "number": "3",
        "title": "Smart Reminders & Backup",
      },
    ],
    "subtitle": "One app that keeps your subs on schedule â€” without more calls or texts.",
  },
  "meta": {
    "lang": "en",
    "title": "BuildFlow - Stop Chasing Subs. Start Building.",
  },
  "noteFromBuilder": {
    "content": [
      "I came from the job siteâ€”loading materials, sweeping floors, doing whatever needed to get done. Within six months, I was running multiple projects, trying to keep subs lined up and jobs moving. Between tracking confirmations, chasing no-shows, and explaining delays to clients, most days felt like managing chaos.",
      "That's when I thought: there had to be a better way.",
      "That's why I built BuildFlow. Not a bloated platform with a thousand tabsâ€”just a simple tool for people like you and me: managing multiple jobs, juggling trades, and keeping the day on track. BuildFlow helps you stay in control, keep subs aligned, and cut through the chaos.",
    ],
    "headline": "Built on the Job Site â€” By Someone Still in It",
    "signature": {
      "name": "â€” Dallas",
      "title": "Superintendent & Creator of BuildFlow",
    },
  },
  "problem": {
    "headline": "Scheduling subs shouldn't take 20 hours a week.",
    "items": [
      {
        "description": "15+ calls a day just to see who's showing up. "You still good for Tuesday?" shouldn't be your full-time job.",
        "title": "Your Phone Became the Job",
      },
      {
        "description": "Subs ghost you, delays cascade, and you're left explaining to clients why their project is behind schedule.",
        "title": "Missed Deadlines",
      },
      {
        "description": "Every hour spent texting is an hour lost on-site or finding new projects.",
        "title": "Lost Revenue",
      },
    ],
  },
  "projectsScreenshot": {
    "headline": "Multiple Jobs, Zero Chaos",
    "subtitle": "Whether you've got two projects or ten, BuildFlow helps you track them allâ€”clearly and simply.",
  },
  "textConversations": {
    "headline": "Real Conversations, Real Simple",
    "subtitle": "Turn no-shows into confirmed jobs with automated texts and reminders.",
  },
}
`;
